plugins/voltageclamp/src/activation.cc

```diff
@@ -62,17 +102,13 @@ int Activation::main( void )
   holdingsignal.constWave( holdingpotential );
   holdingsignal.setIdent( "VC=" + Str( holdingpotential ) + "mV" );
 
-  // stimulus:
-  OutData signal;
-  signal.setTrace( PotentialOutput[0] );
-  // signal.pulseWave( duration, -1.0,  testingpotential, holdingpotential );
-  signal.constWave( duration, -1.0,  testingpotential );
-
-  OutData signal2;
-  signal2.setTrace( PotentialOutput[0] );
-  signal2.pulseWave( duration, -1.0,  20.0, holdingpotential );
+  // clear plot and set Range
```

Comment:
The plot lock needs to come first! When you start the Repro it gets a repaint event which causes it to draw the plots which whatever data the plot had from the previous run of the RePro. While the plot is drawn your RePro could just execute the clearData() lines, i.e. making the data the plot is drawing invalid and thus causing a segmentation fault. By locking the plot the redraw would finish before you clear the data or you first clear the data completely and the the plot gets drawn. It is pure luck that this happens only with the dynamic clamp and both relacs and nix saving anabled. This is how is should look like:

P.lock();
P[0].clearData();
P[1].clearData();
P[1].setXRange(mintest,maxtest);

Check all your RePros for this error!